app/models/concerns/course/lesson_plan/item_todo_concern.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+module Course::LessonPlan::ItemTodoConcern
+  extend ActiveSupport::Concern
+
+  included do
+    after_save do |item|
+      if item.try(:actable_type).try(:constantize).try(:has_todo?) && item.draft_changed?
```

Comment:
This actually breaks abstraction but it will have better performance. 

Not all lesson plan items have the actable object, think it would be better to make this an instance method: `item.has_todo?`